tests/elis_api_client/test_annotations.py

```diff
@@ -239,6 +239,56 @@ def is_imported(annotation):
 
         sleep_mock.assert_called_once_with(2)
 
+    async def test_poll_annotation_until_imported(self, elis_client, dummy_annotation):
+        client, http_client = elis_client
+        in_progress_annotation = {**dummy_annotation, "status": "importing"}
+        # First, return annotation in importing, than to_review state
+        http_client.fetch_one.side_effect = [in_progress_annotation, dummy_annotation]
+        # Return sideloaded content
+        http_client.request_json.return_value = {"content": []}
+
+        with patch("asyncio.sleep") as sleep_mock:
+            annotation = await client.poll_annotation_until_imported(
+                dummy_annotation["id"], sleep_s=2, sideloads=["content"]
+            )
+
+        assert annotation == Annotation(**dummy_annotation)
+        sleep_mock.assert_called_once_with(2)
+
+    async def test_upload_and_wait_until_imported(self, elis_client, dummy_annotation):
+        client, http_client = elis_client
+        in_progress_annotation = {**dummy_annotation, "status": "importing"}
+        # Mock uploading a document
+        http_client.upload.side_effect = [
+            {"results": [{"annotation": f"/annotation/{dummy_annotation['id']}"}]}
+        ]
+        # First, return annotation in importing, than to_review state
+        http_client.fetch_one.side_effect = [in_progress_annotation, dummy_annotation]
+        # Return sideloaded content
+        http_client.request_json.return_value = {"content": []}
+
+        with patch("asyncio.sleep") as sleep_mock:
+            annotation = await client.upload_and_wait_until_imported(
+                queue_id=8199,
+                filepath="tests/data/sample_invoice.pdf",
+                filename="document.pdf",
+                sleep_s=2,
+                sideloads=["content"],
+            )
+
+        assert annotation == Annotation(**dummy_annotation)
+
+        sleep_mock.assert_called_once_with(2)
+
+    async def test_start_annotation(self, elis_client, dummy_annotation):
+        client, http_client = elis_client
+        http_client.replace.return_value = dummy_annotation
+
+        aid = dummy_annotation["id"]
+
+        await client.start_annotation(aid)
+        http_client.request_json.assert_called_with("POST", "annotations/314528/start")
```

Comment:
```suggestion
        http_client.request_json.assert_called_with("POST", f"annotations/{aid}/start")
```

I would prefer this to make it clear where is the number coming from. WDYT?

Reply:
I usually prefer to hardcode the values in asserts but why not in this case...